### ISSUE TITLE
Script hangs on fail.

### ISSUE THREAD

**Issue by kareltucek** [konsumer/tplink-lightbulb#53]
Hi!

If the command (e.g., turning light bulb on) fails (e.g., because network interface is unreachable), tplight does not exit. Instead, it remains hanging around forever.

For instance:
```
$ tplight on 10.0.0.0
Error: send EACCES 10.0.0.0:9999
    at doSend (dgram.js:683:16)
    at defaultTriggerAsyncIdScope (internal/async_hooks.js:323:12)
    at afterDns (dgram.js:629:5)
    at processTicksAndRejections (internal/process/task_queues.js:85:21) {
  errno: 'EACCES',
  code: 'EACCES',
  syscall: 'send',
  address: '10.0.0.0',
  port: 9999
}
```

**Comment by konsumer** [comment]
I think this will need testing but it has to do with promise-catching. For example, [here](https://github.com/konsumer/tplink-lightbulb/blob/master/src/cli.js#L84) is the `catch` for "on".

I think something like this would work:

```js
.catch(err => {
  console.error(err)
  process.exit(1)
})
```

**Comment by kareltucek** [comment]
Thanks for reply and sorry for so late reply on my part!

> I think something like this would work:

Yes I do confirm that this solves the problem.

**Comment by konsumer** [comment]
Should be resolved in 1.5.4
